src/tests/ftest/util/ec_utils.py

```diff
@@ -57,11 +93,11 @@ def ior_param_update(self, oclass, sizes):
             oclass(list): list of the obj class to use with IOR
             sizes(list): Update Transfer, Chunk and Block sizes
         """
+        self.ior_cmd.block_size.update(sizes[1])
```

Comment:
same comment as above (size[1] ---> size["block_size"]

Reply:
Well, this is from YAML file so not sure we need to update this as one line with all sizes looks easy. If we want to add a different size it will be a single line instead of three different lines.

```
   # [ChunkSize, BlocksSize, TransferSize]
   - [32M, 128M, 8M]       #Full Striped
```